apps/builder/app/builder/features/sidebar-left/panels/pages/page-settings.tsx

```diff
@@ -1480,6 +1480,10 @@ const updatePage = (pageId: Page["id"], values: Partial<Values>) => {
       registerFolderChildMutable(folders, page.id, values.parentFolderId);
     }
 
+    if (values.marketplaceInclude !== undefined) {
+      page.marketplace ??= {};
```

Comment:
what's this? ??=

Reply:
modern js
equivalent of
page.marketplace = page.marketplace ?? {};